arangod/IResearch/IResearchInvertedIndexMeta.h

```diff
@@ -23,13 +23,171 @@
 
 #pragma once
 
+#include "IResearchDataStoreMeta.h"
 #include "IResearchLinkMeta.h"
-#include "IResearchViewSort.h"
 #include "IResearchViewStoredValues.h"
 
+#include <unicode/locid.h>
+
 namespace arangodb::iresearch {
 
-struct IResearchInvertedIndexMeta {
+enum class Consistency { kEventual, kImmediate };
+
+class IResearchInvertedIndexSort {
+ public:
+  IResearchInvertedIndexSort() = default;
+  IResearchInvertedIndexSort(const IResearchInvertedIndexSort&) = default;
+  IResearchInvertedIndexSort(IResearchInvertedIndexSort&&) = default;
+  IResearchInvertedIndexSort& operator=(const IResearchInvertedIndexSort&) =
+      default;
+  IResearchInvertedIndexSort& operator=(IResearchInvertedIndexSort&&) = default;
+
+  bool operator==(IResearchInvertedIndexSort const& rhs) const noexcept {
+    return _fields == rhs._fields && _directions == rhs._directions &&
+           std::string_view(_locale.getName()) ==
+               std::string_view(rhs._locale.getName());
+  }
+
+  bool operator!=(IResearchInvertedIndexSort const& rhs) const noexcept {
```

Comment:
AFAIK we don't need it anymore for `c++20`

Reply:
Good point. Done